projects/urls.py

```diff
@@ -9,5 +9,6 @@
     path("admin/", admin.site.urls),
     path("media_server/", include("openedx_learning.contrib.media_server.urls")),
     path("rest_api/", include("openedx_learning.rest_api.urls")),
+    path("tagging/rest_api/", include("openedx_tagging.core.tagging.urls")),
     path('__debug__/', include('debug_toolbar.urls')),
```

Comment:
Ach.. from the [failing test](https://github.com/open-craft/openedx-learning/actions/runs/5603509869/jobs/10250233758?pr=4), it looks like we are stumbling on a missing `debug_toolbar` app in testing.

But that app shouldn't be enabled on a production service, which this line is doing. Could you remove this line please? 
```suggestion
```